Community-Supported/README.md

```diff
@@ -6,6 +6,8 @@ The community samples focus on individual use cases and are Python-only. They ha
 </br>
 
 ## What samples are available?
+- __adjust-vertex-order__
+  - Demonstrates how to adjust the vertex order of all polygons in a `.hyper` file by copying all of the tables and data and calling transformative SQL function on all columns of type `GEOGRAPHY`.
```

Comment:
Can we somehow link to the sample folders? @tsjoblad, can we do this for all samples?

Reply:
yep!